### DISCUSSION TITLE
recipes/cookies-and-bars/snickerdoodles/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#749] [Comments] [answered]
# recipes/cookies-and-bars/snickerdoodles/

A collection of my recipes.

https://nicholaswilde.io/recipes/cookies-and-bars/snickerdoodles/

**Comment by nicholaswilde** [comment]
Age at room temperature for 30 min
30 g at 7 min bake okay.
30 g at 10 min burned
60 g at 8.5 min dark
Turned out a little flat

**Comment by nicholaswilde** [comment, marked as answer]
2nd attempt is at cold dough, 375F, 30g for 9 minutes and came out perfect. Aged 24 hours.
